scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -0,0 +1,213 @@
+import type { AssetId } from '@shapeshiftoss/caip'
+import { FEE_ASSET_IDS, fromAssetId } from '@shapeshiftoss/caip'
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import type { AssetsById } from '@shapeshiftoss/types'
+import assert from 'assert'
+import axios from 'axios'
+import axiosRetry from 'axios-retry'
+import { Presets, SingleBar } from 'cli-progress'
+import fs from 'fs'
+import { isNull } from 'lodash'
+import isUndefined from 'lodash/isUndefined'
+import path from 'path'
+
+import { zerionImplementationToMaybeAssetId } from './mapping'
+import { zerionFungiblesSchema } from './validators/fungible'
+
+const ZERION_BASE_URL = 'https://api.zerion.io/v1'
```

Comment:
Can we add a comment here with the rationale on not using `https://zerion.shapeshift.com` so this doesn't get updated in the future and rugged by rps?

Reply:
e0ebc7869f19cb4eb09a0ce49f62c755fda42ec2